testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/kerberos/AbstractKerberosSingleRealmTest.java

```diff
@@ -131,35 +132,19 @@ public void usernamePasswordLoginTest() throws Exception {
         // Change editMode to READ_ONLY
         updateProviderEditMode(UserStorageProvider.EditMode.READ_ONLY);
 
-        // Login with username/password from kerberos
-        changePasswordPage.open();
-        loginPage.assertCurrent();
-        loginPage.login("jduke", "theduke");
-        changePasswordPage.assertCurrent();
```

Comment:
As this test case is included in the abstract test and is used for various additional test classes, IMHO, the login should be verified here via the LoginPage. Also, the test case is named `usernamePasswordLoginTest`, and one could say if I execute this successfully, I'm really confident the login works as expected.

We're not sure the login with password `theduke` worked before, as it's tested in the following steps.